cypress/support/index.js

```diff
@@ -0,0 +1,9 @@
+Cypress.Commands.add('getByDataCy', (selector) => {
+    cy.get(`[data-cy="${selector}"]`);
+});
+
+Cypress.Commands.add('selectElement', {
+  prevSubject: 'optional'
+}, (subject, attribute, index = 0) => {
+  cy.wrap(subject).eq(index).find(`[data-cy=${attribute}]`);
+});
```

Comment:
```suggestion
});

```
порожній рядок в кінці фалу
також команда selectElement в цій задачі не використовується

Reply:
done